src/test/java/com/org/jenkins/custom/jenkins/distribution/service/DockerBuildTest.java

```diff
@@ -0,0 +1,18 @@
+package com.org.jenkins.custom.jenkins.distribution.service;
+
+import com.palantir.docker.compose.DockerComposeRule;
+import org.junit.ClassRule;
+import org.junit.jupiter.api.Test;
+
+public class DockerBuildTest {
+
+    @ClassRule
+    public static DockerComposeRule docker = DockerComposeRule.builder()
+        .file("src/test/resources/docker-compose.yml")
+        .build();
+
+    @Test
+    public void testDockerBuild() throws Exception {
```

Comment:
This test throws an error saying `cannot access forbidden path`

Reply:
If this is a problem on the Jenkins builds, then you also need to update the `Jenkinsfile` to use agents that can run docker tests.